perfmetrics/scripts/testing_on_gke/examples/fio/loading-test/templates/fio-tester.yaml

```diff
@@ -16,7 +16,7 @@
 apiVersion: v1
 kind: Pod
 metadata:
-  name: fio-tester-{{ .Values.fio.readType }}-{{ lower .Values.fio.fileSize }}-{{ lower .Values.fio.blockSize }}-{{ .Values.scenario }}
+  name: fio-tester-{{ .Values.instanceId }}-{{ .Values.scenario }}-{{ .Values.fio.readType }}-{{ lower .Values.fio.fileSize }}-{{ lower .Values.fio.blockSize }}-{{ .Values.fio.numThreads }}-{{ .Values.fio.filesPerThread }}
```

Comment:
Please refer to my other comment regarding long pod names as it applies here as well.

Reply:
Please see the response in https://github.com/GoogleCloudPlatform/gcsfuse/pull/2359#discussion_r1722956358 .